src/Actions/Family/CreateFamily.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Choredo\Actions\Family;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Zend\Diactoros\Response\EmptyResponse;
+
+class CreateFamily
+{
+    /**
+     * @var EntityManagerInterface
+     */
+    private $entityManager;
+
+    /**
+     * CreateFamily constructor.
+     * @param EntityManagerInterface $entityManager
+     */
+    public function __construct(EntityManagerInterface $entityManager)
+    {
+        $this->entityManager = $entityManager;
+    }
+
+    public function __invoke(ServerRequestInterface $request, ResponseInterface $response)
+    {
+        $family = $request->getAttribute('familyEntity');
+
+        $this->entityManager->persist($family);
+        $this->entityManager->flush();
+
+        return EmptyResponse::withHeaders([
```

Comment:
Is this returning a redirect? Shouldn't it just return the transformed family?

Reply:
Spec says 201 with the Location header set to the URI of the new resource, where applicable. A response entity describing status is optional. We could use it to return the created ID (either alone or with the rest of the entity). Up to you, whatever makes the FE simpler.

Reply:
Per our convo, let's return 201 with the entity, consistent with JsonAPI Spec, por favor. <3 <3